run_pseudobulk_benchmark.py

```diff
@@ -44,26 +46,26 @@
     # adata = scvi.data.heart_cell_atlas_subsampled()
     # preprocess_scrna(adata, keep_genes=1200)
 elif BENCHMARK_DATASET == "CTI":
-    adata = sc.read("/home/owkin/project/cti/cti_adata.h5ad")
-    adata, filtered_genes = preprocess_scrna(adata,
-                     keep_genes=N_GENES,
-                     batch_key="donor_id")
+    cti_path = f"/home/owkin/data/cti_data/processed/cti_processed_{N_GENES}.h5ad"
+    if os.path.exists(cti_path):
+        adata = sc.read(f"/home/owkin/data/cti_data/processed/cti_processed_{N_GENES}.h5ad")
```

Comment:
You forgot the project folder, it should be `/home/owkin/project/data/cti_data/processed/cti_processed_{N_GENES}.h5ad`